_data/sponsors.yml

```diff
@@ -85,6 +85,29 @@ sponsors:
       team</a>, building new products, and scaling our systems to meet a >20x surge
       in demand during the next 12-18 months.
        </p>
+  - name: capitec
+    active: true
+    active_rank: 3
+    website_text: >
+      Capitec is a South African retail bank that offers simplified,
+      affordable and transparent banking using innovative technology.
+      </p>
+      <p>
+      Our driving force is to help our clients improve their financial lives and put them in control of their
+      money. We do this through our Global One solution, which offers our clients transactions at lower
+      fees, savings plans with highly competitive interest rates, personalised credit products and an
+      affordable funeral plan based on their needs. And with technology like our cellphone banking app and
+      tap to pay-enabled Global One card, we put the value back into our clients’ pockets – helping them to
+      simplify banking and live better.
+      </p>
+      <p>
+      This commitment, combined with our pillars of simplicity, affordability, accessibility and personalised
+      experience, has not only earned us the title of South Africa’s leading retail bank, but also recognition
+      as the best bank in the world.
+      </p>
+      <p>
+      Visit <a href='https://www.capitecbank.co.za/'>https://www.capitecbank.co.za/</a> for more info.
```

Comment:
Usually I make it <a href='https://www.capitecbank.co.za/'>www.capitecbank.co.za</a> just for consistency with the others.